tutorials/combine-feature-collections/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Combine FeatureCollections into a new FeatureCollection
+description: This basic tutorial shows how users can combine two FeatureCollections into one. It is targeted at beginners.
+author: sabrinaszeto
+tags: beginner, featurecollection
+date_published: 2019-09-18
```

Comment:
Why is each line prefixed by "+"? I saw this in @TC25's PR as well. Ws this by accident or intentional?